xtask/src/util.rs

```diff
@@ -411,3 +413,12 @@ pub fn write_to_string(path: &Path, contents: &str) -> cross::Result<()> {
     writeln!(file, "{}", contents)?;
     Ok(())
 }
+
+// https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files
+pub fn write_to_gha_env_file(env_name: &str, contents: &str) -> cross::Result<()> {
+    let path = env::var(env_name)?;
+    let path = Path::new(&path);
+    let mut file = fs::OpenOptions::new().write(true).open(path)?;
```

Comment:
this needs to append, not write over.

```suggestion
    let mut file = fs::OpenOptions::new().append(true).open(path)?;
```